provision/minikube/keycloak/templates/keycloak.yaml

```
@@ -26,7 +26,11 @@ spec:
   db:
 {{ if or (eq .Values.database "aurora-postgres") (eq .Values.database "postgres") (eq .Values.database "postgres+infinispan") }}
     vendor: postgres
+{{- if .Values.useAWSJDBCWrapper}}
+    url: jdbc:aws-wrapper:postgresql://postgres:5432/keycloak
```

I'm not sure if we should recommend using the external service to connect to Aurora. Instead, I'd prefer to use the direct DNS name of the database. This would prevent potential DNS obscurities due to indirection, and the JDBC wrapper might have direct access to multiple DNS entries. When doing this, `aurora-service.yaml` would no longer be needed.
 

```suggestion
    url: jdbc:aws-wrapper:postgresql://{{ .Values.dbUrl }}:5432/keycloak
```